src/components/Popover/index.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { Drop, DropProps } from 'grommet';
+import styled from 'styled-components';
+import { px } from '../../utils';
+import TooltipArrow, { TooltipArrowProps } from '../../internal/TooltipArrow';
+
+const StyledDrop = styled(Drop)`
+	border: 0;
+`;
+
+const StyledDropInner = styled.div`
```

Comment:
Do we need to set a `max-width` here? What happens if the content is huge? Maybe `overflow` as well?